extensions/wikia/AbTesting/js/AbTest.js

```diff
@@ -207,6 +207,9 @@
 	// Returns true if value falls into provided ranges
 	function isInRanges( value, ranges ) {
 		var i, range;
+		if ( !ranges || !ranges.length ) {
```

Comment:
I've just run grep and it seems we rarely use `Array.isArray` blindly. I'll stick with the current code.